### PR TITLE
Update the submodule pointer to include the new lookup operator integration tests

### PR DESCRIPTION
**Pull request by Dakostu**
This change is another submodule pointer bump.

### CODE REVIEW

File: contrib/tenzir-plugins
Comment (jachris) [inline contrib/tenzir-plugins:1]: Can you rebase the PR so that CI can run? Thanks!

Comment (Dakostu) [reply contrib/tenzir-plugins:1]: Done.

**Review by jachris — APPROVED** [review]
Not sure why this was stuck so long in the review-limbo, sorry for that. LGTM!

**Comment by jachris** [comment]
Don't forget to rebase the plugin PR and update this one here for CI to pass.

**Pull request merged**